### ISSUE TITLE
[Bug] events.out.tfevents is not generated when using --restore_path  flag in python TTS/TTS/bin/train_tacotron.py

### ISSUE THREAD

**Issue by Sadam1195** [coqui-ai/TTS#563]
**Describe the bug**
Each training run is supposed to generate `events.out.tfevents` for tensorboard but when I am using` --restore_path` it doesn't generate` events.out.tfeventsxxxx` file

**To Reproduce**
1. Run the following command 
```
 CUDA_VISIBLE_DEVICES="0" python TTS/TTS/bin/train_tacotron.py --restore_path ./Results/ljspeech-ddc-June-09-2021_08+36PM-0000000/best_model.pth.tar \
                                                         --config_path ./tacotron2-DDC.json \
                                                          --coqpit.output_path ./Results  \
                                                          --coqpit.datasets.0.path ./Italian_dataset/it_IT/by_book/male/riccardo_fasol/il_ritratto_del_diavolo   \
                                                          --coqpit.audio.stats_path ./scale_stats.npy \
```
2. **No Runtime or compile Error**

**Expected behavior**
It should generate ` events.out.tfeventsxxxx` file in the training RUN directory by default. 

**Environment (please complete the following information):**
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:
- Ubuntu 18.04.5 LTS
- **PyTorch or TensorFlow version (use command below)**:
- PyTorch 1.4.0, TensorFlow 2.1.0

- **Exact command to reproduce**:
```
 CUDA_VISIBLE_DEVICES="0" python TTS/TTS/bin/train_tacotron.py --restore_path ./Results/ljspeech-ddc-June-09-2021_08+36PM-0000000/best_model.pth.tar \
                                                         --config_path ./tacotron2-DDC.json \
                                                          --coqpit.output_path ./Results  \
                                                          --coqpit.datasets.0.path ./Italian_dataset/it_IT/by_book/male/riccardo_fasol/il_ritratto_del_diavolo   \
                                                          --coqpit.audio.stats_path ./scale_stats.npy \
```

**Comment by erogol** [comment]
when you call `--restore_path`, it starts a new training in a different folder with the restored model. Did you check this new folder?

**Comment by Sadam1195** [comment]
> when you call `--restore_path`, it starts a new training in a different folder with the restored model. Did you check this new folder?

Yes, I did. New folder had all other files/folders except for events.out.tfevents file.

**Comment by Sadam1195** [comment]
> > when you call `--restore_path`, it starts a new training in a different folder with the restored model. Did you check this new folder?
> 
> Yes, I did. New folder had all other files/folders except for events.out.tfevents file.

Just to make sure I will run the command again to see if the problem is consistent. If the issue appears, I will reopen the issue. I am closing it for now.

**Comment by Ca-ressemble-a-du-fake** [comment]
Hi,
It seems that the `events.out.tfevents` file is not updated when continuing training VITS model. And even while training from scratch the behaviour is odd because sometimes events.out file gets only updated once after 70k steps (and then you can eg listen to test sentences at many points in time).

**Comment by Ca-ressemble-a-du-fake** [comment]
That's weird! Looking at the folder now just after writing the above comment a new `events.out.tfevents` file has just appeared (1 day after starting the continue training). File size is 569MB and creation date is from yesterday. Maybe `events.out.tfevents` file is only output when its size is big enough ? So adding many test sentences would make it grow faster and be updated oftener ?

**Comment by Sadam1195** [comment]
> Maybe events.out.tfevents file is only output when its size is big enough ?

When I last trained models on colab I noticed my io operations limit maxed out so `events.out.tfevents` file writing in real-time wasn't being placed on drive. So until training gets interrupted or stops only then updated `events.out.tfevents` was being written on drive. Maybe it's `tf` bug but AFAIK in my case it was due to Colab's i/o operation bound limits being maxed out.

@Ca-ressemble-a-du-fake
